sig-community/charter.md

```diff
@@ -0,0 +1,60 @@
+# SIG Community
+
+## Scope
+
+This SIG cares about all aspects of what is helpful and harmful to this open source software community.
+We understand that this project is a human endeavor, and it is our job to put people before practice, process, and even principle.
+
+**What does it mean to support a community?**
+It means understanding and putting resources into whatever is important to people in this community.
+We put our own time and energy into this support, and we help coordinate and direct project resources.
+
+**What is an open source project compared to a community?**
+A _project_ is a general term referring to an effort by people to accomplish something that requires time and multiple steps. Pruning a rose bush is a task, where a multi-year effort to plant a new rose garden is a project.
+An open source software community is a type of _community of practice_, and the software project is the place where the community practices.
+
+You can have a community without a project, which is really a social club.
+But you cannot have an open source project without a community.
+However, not all open source software is maintained by a project.
+It may rather be maintained by an individual, or a single organization by fiat.
+
+
+
+### In scope
+
```

Comment:
Should we put as 1st item, that we want to establish a wide, diverse and inclusive community  from all backgrounds and vendors?

Reply:
```suggestion

* Establishing a wide, diverse, and inclusive community of people and organizations from all backgrounds and situations.
```